webapp/src/components/CreateMeetingNoteForm.tsx

```diff
@@ -0,0 +1,112 @@
+import { Alert, Card, CardContent, Snackbar, TextField } from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useState, useContext } from 'react';
+
+import { EntityId } from '../types/api';
+import SessionContext from './SessionContext';
+
+interface CreateMeetingNoteFormProps {
+  meetingId: EntityId;
+  onMeetingNoteChanged?: (id: EntityId) => void;
+  greatestSortOrderOfMeetingNotesList: number;
+}
+
+const CreateMeetingNoteForm = ({
+  meetingId,
+  onMeetingNoteChanged,
+  greatestSortOrderOfMeetingNotesList,
+}: CreateMeetingNoteFormProps) => {
+  const { principalId } = useContext(SessionContext);
+
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+  const [isSuccessMessageVisible, setIsSuccessMessageVisible] = useState(false);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const onSubmit: FormEventHandler = event => {
+    event.preventDefault();
+    setIsSavingMeetingNote(true);
+
```

Comment:
👾

Reply:
Curious what does the emoji mean in this context? Is that alien emoji meaning for a bug? Or a foreign object?

Reply:
It's the "space invader" emoji. I'm just being silly. There's inconsistent white space here, so I thought I'd drop a subtle hint to manage white space more carefully.

Reply:
Thanks for clarifying!